v-next/hardhat/src/internal/builtin-plugins/solidity-test/config.ts

```diff
@@ -121,6 +121,11 @@ export async function resolveSolidityTestUserConfig(
   testsPath = typeof testsPath === "object" ? testsPath.solidity : testsPath;
   testsPath ??= "test";
 
+  const solidityTest = {
+    rpcCachePath: "edr-cache",
```

Comment:
This should be inside `config.paths.cache`. A subfolder I mean.

There should be a single cache directory in the root of the project.